### ISSUE TITLE
Content provider's content file

### ISSUE THREAD

**Issue by baloyan** [baloyan/cachecash-demo#4]
Current issue owner should work close with issue #1 
What we want ?

1. We want at least 10 video file and they should be mp4 format, two video file you can find here https://github.com/baloyan/cachecash-demo/tree/master/website/files
So, you should find a video which you think worth to show during CacheCash demo, like one of videos is Mars Rover Spirit Launch 

2. We want to merge all video in one file and name it `all_in_one.mp4` 

3. We want to get specific code type of  `all_in_one.mp4` video.
   You can use mp4info tool for that, see `./mp4info frag_bunny.mp4 | grep Codec`

**Comment by GhadaAlmashaqbeh** [comment]
I do not think this is a task, Justin even said the same in the meeting. It needs at max 10 min to be done.

**Comment by awwad** [comment]
As a sidenote, on OS X:

```bash
brew install media-info
mediainfo <video file>
```
